projects/js-packages/publicize-components/src/utils/test-constants.js

```diff
@@ -9,6 +9,16 @@ export const SUPPORTED_SERVICES_MOCK = [
 			additional_users_only: true,
 		},
 	},
+	{
+		id: 'linkedin',
+		label: 'LinkedIn',
+		description: 'Share to your account and company pages',
+		url: 'https://test_url.com',
+		supports: {
+			additional_users: true,
+			additional_users_only: false,
+		},
+	},
```

Comment:
Shouldn't this file live with the tests?

Reply:
It's shared between different tests.